README.md

```diff
@@ -7,3 +7,25 @@ The ORCESTRA campaign is described on [this website](https://orcestra-campaign.o
 
 Kuettner, J. P.: General description and central program of GATE, Bulletin of the
 American Meteorological Society, 55, 526–530, 1974.
+
+### Using PAMTRA for the sondes
+
+This package is build using uv. Using the `uv.lock` is sufficient to run most of the python plotting scripts. However, to use the pamtra model, it has to be installed separately. On levante, the following steps suffice to do that, if run from within the repo.
+
+```
+module load git
+module load netcdf-c
+module load fftw
+module load gcc
+spack load /tpmfvwu # openblas
+
+CC=gcc uv run pip install git+https://github.com/igmk/pamtra
```

Comment:
```suggestion
uv run pip install git+https://github.com/igmk/pamtra
```
This should actually work without specifying the compiler **if** you don't load the `python3` module